<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(191, 208, 188)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_picdel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string>‚ùå</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Delete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
	background-color: rgb(255, 0, 0);
}
QPushButton{background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Display">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>630</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
background-color: rgb(0, 170, 0);
}
QPushButton{background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);}</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_picdisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string>üó®</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>691</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete &amp; Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Add">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
background-color: rgb(0, 170, 0);
}
QPushButton{background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_bid">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Title">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Borrow_by">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_borrow_date">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Due">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bid">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>115</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Borro_by">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Borrow By</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Bdate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Borrow Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_due_date">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Due Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_picadd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string>‚úî</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
